src/templates/index.html

```diff
@@ -344,6 +356,230 @@ <h2 class="info-title">Paper Info:</h2>
         </div>
     </div>
 
+    <!--TUTORIAL PAGE-->
+    <div id="tutorial-page" class="hidden">
+        <div id="tutorial-text">
+            <h1 id="overview">Getting Started</h1>
+            <p>PDFLegacy is a tool to provide better reading experiences for researchers. This PDF reader has four key
+                features:
+            <ul>
+                <li>Extract essential information (such as
+                    figures, references, etc.) from the paper.
+                </li>
+                <li>
+                    Mine knowledge from the extracted information to help with paper comprehension and inspire new
+                    research
+                    ideas by looking at correlated papers.
+                </li>
+                <li>
+                    Generating/mining summarizations for key components of the paper
+                </li>
+                <li>
+                    Highlighting commonly used, topic-specific phrasings
+                </li>
```

Comment:
Some of these have a period (.) at the end and some not - choose one or the other and stick to it, this looks a bit messy... Maybe without since theyre in annotation form?

Reply:
fixed (without)